packages/token-search-discovery-controller/src/types.ts

```diff
@@ -16,7 +29,7 @@ export type TokenSearchResponseItem = {
   logoUrl?: string;
 };
 
-export type TokenTrendingResponseItem = {
+export type MoralisTokenResponseItem = {
```

Comment:
Nitpick: this seems unrelated to a vanilla Response object and looks to be only the tokenItem. Adding Response into the name confuses me a bit and could lead people to think this includes status and other properties of an HTTP Response

https://developer.mozilla.org/en-US/docs/Web/API/Response

Reply:
Recommend dropping the ResponseItem portion all together to simply be `MoralisToken`?  I like that too.